pkg/internal/sqlsmith/schema.go

```diff
@@ -57,13 +62,31 @@ func (s *Smither) ReloadSchemas() error {
 	return err
 }
 
-func (s *Smither) getRandTable() (*tableRef, bool) {
+func (s *Smither) getRandTable() (*aliasedTableRef, bool) {
 	s.lock.RLock()
 	defer s.lock.RUnlock()
 	if len(s.tables) == 0 {
 		return nil, false
 	}
-	return s.tables[s.rnd.Intn(len(s.tables))], true
+	table := s.tables[s.rnd.Intn(len(s.tables))]
+	indexes := s.getIndexes(*table.TableName)
+	var indexFlags tree.IndexFlags
+	if s.coin() {
+		indexNames := make([]tree.Name, 0, len(indexes))
+		for _, index := range indexes {
+			if !index.Inverted && !strings.Contains(index.Name.String(), "jsonb") {
```

Comment:
do you need the second condition here? you can't make a non-inverted jsonb index, only inverted.